src/bilder/images/edxapp/templates/common_values.yml

```diff
@@ -377,8 +378,14 @@ PLATFORM_TWITTER_ACCOUNT: '@YourPlatformTwitterAccount'
 POLICY_CHANGE_GRADES_ROUTING_KEY: edx.lms.core.default
 PRESS_EMAIL: support@mitxonline.mit.edu # MODIFIED
 PROCTORING_BACKENDS:
-    DEFAULT: 'null'
-    'null': {}
+    DEFAULT: 'proctortrack'
+    'proctortrack':
+      {{ with secret "secret-mitxonline/edxapp" }}
+      'client_id': '{{ .Data.proctortrack-client-id }}'
+      'client_secret': '{{ . Data.proctortrack-client-secret }}'
```

Comment:
The extra space between `.` and `Data` will be problematic. Also, you'll want to add those values to the SOPS file to be populated in Vault via Pulumi.